tests/sweep_framework/sweeps/eltwise/ternary/lerp/lerp.py

```diff
@@ -42,6 +42,18 @@
 }
 
 
+# Invalidate vector is called during the generation phase where each vector will be passed in.
+# If invalidated, the vector will still be stored but will be skipped.
+# Returns False, None if the vector is valid, and True, str with a reason for invalidation if it is invalid.
+def invalidate_vector(test_vector) -> Tuple[bool, Optional[str]]:
+    if (
+        test_vector["input_b_dtype"] != test_vector["input_a_dtype"]
+        or test_vector["input_c_dtype"] != test_vector["input_a_dtype"]
+    ):
```

Comment:
```suggestion
    if test_vector["input_a_dtype"] not in (test_vector["input_b_dtype"], test_vector["input_c_dtype"]):
```

Reply:
Actually, both the input tensors b and c should be of the same type as input tensor a, hence used relational operator